components/tag-list/test/tag-list.test.js

```diff
@@ -133,16 +133,5 @@ describe('d2l-tag-list-item-mixin-consumer', () => {
 			const child = elem.children[3];
 			expect(child._plainText).to.be.equal('Tag');
 		});
-
-		it('should error if not provided', async() => {
-			let error;
-			try {
-				await fixture(html`<d2l-tag-list-item-mixin-consumer></d2l-tag-list-item-mixin-consumer>`);
-			} catch (e) {
-				error = e;
-			}
-			expect(error?.name).to.equal('TypeError');
-		});
```

Comment:
Unfortunately this means we can't test for it any more.

I'm going to see if I can come up with a more reusable/consistent way to validate required things like this, and in doing so maybe make it easier to test for them as well.